bip_metainfo/src/builder/mod.rs

```diff
@@ -161,6 +173,57 @@ impl<'a> MetainfoBuilder<'a> {
         self
     }
 
+    /// Get decoded value of announce-list key
+    pub fn get_trackers(self) -> Option<Vec<Vec<String>>> {
+        let dict_access = self.root.dict().unwrap();
+
+        parse::parse_announce_list(dict_access)
+            .and_then(|list| {
```

Comment:
Are you able to replace the .and_then and .or with a single .map?

Eg ```parse::parse_announce_list(dict_access).map(parse::convert_announce_list)``` ?

Same for the other getters.

Reply:
done